package.json

```diff
@@ -2,19 +2,22 @@
   "name": "solc",
   "version": "0.8.11",
   "description": "Solidity compiler",
-  "main": "index.js",
+  "main": "dist/index.js",
   "bin": {
-    "solcjs": "solcjs"
+    "solcjs": "dist/solc.js"
   },
   "scripts": {
-    "lint": "eslint .",
-    "lint:fix": "eslint --fix .",
-    "updateBinary": "node downloadCurrentVersion.js && node verifyVersion.js",
-    "prepublishOnly": "npm run updateBinary",
-    "pretest": "npm run lint",
-    "test": "tape ./test/index.js",
-    "coverage": "node ./node_modules/nyc/bin/nyc.js --reporter=lcov --reporter=text-summary ./node_modules/tape/bin/tape ./test/index.js",
-    "coveralls": "npm run coverage && node ./node_modules/coveralls/bin/coveralls.js <coverage/lcov.info"
+    "build": "tsc",
+    "postbuild": "node build/postbuild.js",
+    "lint": "eslint --ext .js,.ts .",
+    "lint:fix": "eslint --fix --ext .js,.ts .",
+    "updateBinary": "node build/clean.js && ts-node ./downloadCurrentVersion.ts && ts-node ./verifyVersion.ts",
+    "prepublishOnly": "npm run updateBinary npm && npm run build",
+    "copyTestFiles": "cp -r ./test/resources ./dist/test/",
+    "pretest": "npm run lint && npm run build && npm run copyTestFiles",
+    "test": "cd dist && tape ./test/index.js",
+    "coverage": "nyc npm run test",
+    "coveralls": "npm run coverage && coveralls <coverage/lcov.info"
```

Comment:
I remember these things not working properly on CI without the `./node_modules/...` paths. What changed to allow the removal of that workaround?

Reply:
For as long as I remember this has not been an issue anymore.

Reply:
This was added back around 2-3 years ago because CI struggled otherwise.